docs/UserGuide.md

```diff
@@ -66,95 +133,125 @@ adding, editing, deleting or viewing of friends in your gitGud friend list.
 
 Links a game and the associated in-game username for the game to a friend with the provided `FRIEND_ID`.
+A link between a particular friend and game shows that the friend plays the game. 
+This association is required for subsequent commands, e.g. for the application to display all games that a friend plays 
+or to recommend a friend to play with based on a particular game.
 
-A link between a particular friend and game shows that the friend plays the game. This association is required for subsequent commands, e.g. for the application to display all games that a friend plays or to recommend a friend to play with based on a particular game.
+<div markdown="block" class="alert alert-info">
+:information_source: If you link an already linked game to particular friend again, the previous link will be 
+overridden with the new link with the new `IN_GAME_USERNAME` and the friend's skill level set to `0`. 
+</div>
 
 Format: `friend --link FRIEND_ID -g GAME_ID -u IN_GAME_USERNAME`
 
-* Both `FRIEND_ID` and `GAME_ID` has to already be inside the friends and games list respectively.
+Command constraints: 
+* Both `FRIEND_ID` and `GAME_ID` have to already exist within the friends and games list respectively.
 
 Examples:
 * `friend --link Draco -g DOTA -u Draco995`
   Links a single game, “DOTA” with the in-game username ‘Draco995’,
   to the friend with the gitGud `FRIEND_ID` ‘Draco’.
 
-Snapshot:
+Screenshot:
 
-![Ui](images/ui-snapshots/friend-link.png)
-<br><ins>gitGud after linking a friend to a game</ins>
+![Friend link Screenshot](images/ui-screenshots/friend-link.png)
+<br><center><ins>Image: gitGud after linking a friend to a game</ins></center>
 
 #### Unlinking a game from a friend: `friend --unlink`
 
 Removes the link between a friend and a game.
+This may be used if for example, a friend no longer plays a previously linked game. 
 
 Format: `friend --unlink FRIEND_ID -g GAME_ID`
 
+Command constraints: 
 * `FRIEND_ID` has to already be inside the friends list.
 * The friend has to be previously linked to the game with the provided `GAME_ID`.
 
 Examples:
 * `friend --unlink Draco -g DOTA` Removes the link between the friend with FRIEND_D 'Draco' and the game with `GAME_ID` 'DOTA'. 'Draco' is now no longer associated with 'DOTA'.
 
-Snapshot:
+Screenshot:
 
-![Ui](images/ui-snapshots/friend-unlink.png)
-<br><ins>gitGud after unlinking a friend from a game</ins>
+![Friend unlink screenshot](images/ui-screenshots/friend-unlink.png)
+<br><center><ins>Image: gitGud after unlinking a game `DOTA' from friend `Draco`</ins></center>
 
 #### Assigning a skill value for a linked game : `friend --skill`
 
 Assigns the provided skill value to the friend with the given `FRIEND_ID` for the linked game with the given `GAME_ID`.
-This command allows you to record your perceived gauge of a friend's skill level at a particular game. 
+This command allows you to record your personal gauge of a friend's skill level at a particular game.
```

Comment:
Consider mentioning somewhere that skill is initialised with the value of 0.

Reply:
good catch, thanks